helix-rest/src/main/java/org/apache/helix/rest/common/HelixDataAccessorWrapper.java

```diff
@@ -14,7 +14,7 @@
  * The caches is of the value from get methods and short lived for the lifecycle of one rest request
```

Comment:
This description is a little confusing. Say:

This is a wrapper for {@link ZKHelixDataAccessor} that caches the result of the batch reads it performs. Note that the usage of this object is valid for one REST request.